packages/expo-image/android/src/main/java/expo/modules/image/ThumbnailRequestCoordinatorExtension.kt

```diff
@@ -1,18 +1,21 @@
 package expo.modules.image
 
+import android.util.Log
 import com.bumptech.glide.request.Request
 import com.bumptech.glide.request.ThumbnailRequestCoordinator
 
 fun ThumbnailRequestCoordinator.getPrivateFullRequest(): Request? {
-  return getPrivateRequest("full")
+  return getPrivateFiled("full")
```

Comment:
is this a small typo or "filed" is intentional?

Reply:
No, thanks for pointing that out. I'll fix that on the main ;)